docs/index.md

```diff
@@ -48,6 +48,28 @@ and i386 builds.
 
 [LLVM]: https://llvm.org
 
+
+## Project history
+OSS-Fuzz was launched in 2016 in response to the
+[Heartbleed](https://heartbleed.com/) vulnerability, discovered in one of the
+most popular open source projects for encrypting web traffic. The vulnerability
+had the potential to affect almost every internet user, yet was caused by a
+relatively simple memory buffer overflow bug that could have been detected by
+fuzzing—that is, by running the code on randomized inputs to intentionally cause
+unexpected behaviors or crashes that signal bugs. At the time, though, fuzzing
+was not widely used and was cumbersome for developers, requiring extensive
+manual effort.
+
+Google created OSS-Fuzz to fill this gap: it's a free service that runs fuzzers
+for open source projects and privately alerts developers to the bugs detected.
+Since its launch, OSS-Fuzz has become a critical service for the open source
```

Comment:
We should update that trophies section below with the counts, and find a way to deduplicate (do we need numbers in both places) or just one).

Reply:
Done